warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__shapes_file_present.sql

```diff
@@ -14,19 +16,28 @@ daily_feed_shapes_present AS (
        AND feed_key IS NOT null
 ),
 
+check_start AS (
+    SELECT MIN(dt) AS first_check_date
+    FROM keyed_parse_outcomes
+),
```

Comment:
why is this different from the others?

Reply:
Discussed offline, the start date is based on whatever underlying data drives the check so it varies from check to check. There's no upstream table with a `_feed_valid_from` column in this table.